platform/pulpcore/app/models/progress.py

```diff
@@ -60,7 +60,7 @@ class ProgressReport(Model):
     total = models.IntegerField(null=True)
     done = models.IntegerField(default=0)
 
-    task = models.ForeignKey("Task", related_name='progress_reports', on_delete=models.CASCADE)
```

Comment:
I think you unintentionally made this change during the rebase to 3.0-dev. The related_name is needed there.